examples/SerialImprov/test_improv.py

```
@@ -0,0 +1,250 @@
+            del buffer[: start + 1]
+            continue
+        packets.append((pkt_type, payload))
+        del buffer[: end + 1]
+    return packets
+
+
+def decode_rpc_result(payload: bytes) -> Tuple[int, list[str]] | None:
+    if len(payload) < 2:
+        return None
+    command = payload[0]
+    total_len = payload[1]
```

Ruff (0.14.1)

Local variable `total_len` is assigned to but never used

Remove assignment to unused variable `total_len`

(F841)